vetiver/tests/test_sklearn.py

```diff
@@ -9,13 +9,13 @@ def _start_application(save_ptype: bool = True):
     model = mock.get_mock_model().fit(X, y)
     v = VetiverModel(
         model=model,
-        ptype_data=X if save_ptype else None,
+        prototype_data=X if save_ptype else None,
```

Comment:
The `save_ptype` arg changed as well, right?

Reply:
Yep! That did change, this is the argument I am using in the pytest fixture, not the actual function. But, I'll update it to be more clear 👍